Assets/SEE/Game/Avatars/PlayerNameMap.cs

```diff
@@ -48,15 +48,27 @@ public static void AddOrUpdatePlayerName(ulong networkObjectId, string playerNam
         /// <returns>corresponding playerName or "Unknown"</returns>
         public static string GetPlayerName(ulong networkObjectId)
         {
+            Log($"{nameof(GetPlayerName)}(networkObjectId={networkObjectId})\n");
+
             if (playerNames.TryGetValue(networkObjectId, out string playerName))
             {
                 return playerName;
             }
             else
             {
                 Debug.LogError($"Player name for player {networkObjectId} is unknown.\n");
+                Dump();
                 return "Unknown";
             }
+
+            void Dump()
+            {
+                Log("Content:\n");
+                foreach (var kvp in playerNames)
+                {
+                    Log($"  {kvp.Key} => {kvp.Value}\n");
+                }
```

Comment:
We can use a single log call for this to avoid cluttering up the log with individual messages:
```suggestion
                string content = string.Join('\n', playerNames.Select(x => $"  {x.Key} => {x.Value}\n"));
                Log("Content:\n" + content);
```